resources/cluster-essentials/templates/addons-configuration.crd.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   creationTimestamp: null
-  labels:
-    controller-tools.k8s.io: "1.0"
   name: addonsconfigurations.addons.kyma-project.io
+  annotations:
+    helm.sh/hook: crd-install
```

Comment:
are you sure about this? other crds definitions don't have it

Reply:
I saw that, and have that should not cause any problem but I can remove it